packages/ERTP/test/unitTests/test-issuerObj.js

```diff
@@ -1,78 +1,59 @@
-// eslint-disable-next-line import/no-extraneous-dependencies
-import '@agoric/install-ses';
-// eslint-disable-next-line import/no-extraneous-dependencies
-import test from 'tape-promise/tape';
+import test from 'ava';
 import { E } from '@agoric/eventual-send';
 import { MathKind, makeIssuerKit } from '../../src';
 
 test('issuer.getBrand, brand.isMyIssuer', t => {
-  try {
-    const { issuer, brand } = makeIssuerKit('fungible');
-    const myBrand = issuer.getBrand();
-    t.ok(myBrand.isMyIssuer(issuer));
-    t.equals(
-      brand,
-      myBrand,
-      'brand returned from `makeIssuerKit` and from `getBrand` the same',
-    );
-    t.equals(issuer.getAllegedName(), myBrand.getAllegedName());
-    t.equals(issuer.getAllegedName(), 'fungible');
-  } catch (e) {
-    t.assert(false, e);
-  } finally {
-    t.end();
-  }
+  const { issuer, brand } = makeIssuerKit('fungible');
+  const myBrand = issuer.getBrand();
+  t.assert(myBrand.isMyIssuer(issuer));
+  t.is(
+    brand,
+    myBrand,
+    'brand returned from `makeIssuerKit` and from `getBrand` the same',
+  );
+  t.is(issuer.getAllegedName(), myBrand.getAllegedName());
+  t.is(issuer.getAllegedName(), 'fungible');
 });
 
-test('amountMath from makeIssuerKit', t => {
-  try {
-    const { amountMath, brand } = makeIssuerKit('fungible');
-    const fungible = amountMath.make;
-    t.ok(
-      amountMath.isEqual(
-        amountMath.add(fungible(100), fungible(50)),
-        fungible(150),
-      ),
-    );
-    t.equals(fungible(4000).value, 4000);
-    t.equals(fungible(0).brand, brand);
-  } catch (e) {
-    t.assert(false, e);
-  } finally {
-    t.end();
-  }
+test('issuer.getAmountMath', async t => {
```

Comment:
I think this is still the same problem that I commented on in the last PR. The original test was testing that the amountMath returned as part of the issuerKit, acted like a real amountMath. 

This new test 1) changed the name to something which isn't right, 2) is testing brands for some reason. Can we keep the same name and test?

Reply:
It was a new test on master that collided with a merge: "amountMath from makeIssuerKit"